strings-csharp/RemoveWhitespaceCharactersFromString/RemoveWhitespaceCharactersFromString/RemoveWhitespaceMethods.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+
+namespace RemoveWhitespaceCharactersFromString
+{
+    public class RemoveWhitespaceMethods
+    {
+        public static string RemoveWhitespacesUsingRegex(string source)
+        {
+            return Regex.Replace(source, @"\s+", String.Empty);
+        }
+
+        public static string RemoveWhitespacesUsingLinq(string source)
+        {
+            return string.Concat(source.Where(c => !char.IsWhiteSpace(c)));
+        }
+
+        public static string RemoveWhitespacesUsingReplace(string source)
+        {
+            return source.Replace(" ", string.Empty);
```

Comment:
Please rename the method to `RemoveSpacesUsingReplace()`. Because it doesn't replace all whitespace but just the space chars?